test/vm.js

```diff
@@ -125,7 +125,7 @@ describe('VM', function() {
       assert.strictEqual(
         vm.url,
         [
-          'https://www.googleapis.com/compute/v1/projects',
```

Comment:
what is this URL? if it's a resource ID it should be `www.googleapis.com/compute/...`